translate.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bytes"
+	"regexp"
+	"strings"
+)
+
+type Redirect struct {
+	Location string
+	Status   int
+}
+
+func Translate(uri string, config *Config) *Redirect {
+	if uri == "" {
+		return nil
+	}
+	if config == nil {
+		return nil
+	}
+	if config.To == "" {
+		return nil
+	}
+
+	redirect := new(Redirect)
+	redirect.Location = config.To
```

Comment:
`go test` shows `cannot use redirect (type Redirect) as type *Redirect in return argument`

Probably should be returning errors instead of `nil` when the TXT records can't be parsed/translated?